django-cloudlaunch/cloudlaunch/backend_plugins/base_vm_app.py

```diff
@@ -313,13 +313,13 @@ def health_check(self, provider, deployment):
         log.debug("Health check for deployment %s", deployment)
         iid = self._get_deployment_iid(deployment)
         if not iid:
-            return {"instance_status": "unknown"}
+            return {"instance_status": "deployment_not_found"}
         log.debug("Checking the status of instance %s", iid)
         inst = provider.compute.instances.get(iid)
         if inst:
             return {"instance_status": inst.state}
         else:
-            return {"instance_status": "deleted"}
+            return {"instance_status": "not_found"}
```

Comment:
It kind of feels that given this is being overloaded and seeing `deployment_not_found`, the value here should be `instance_not_found` as it would make reading it more clear but in the context of the key name, it's probably repetitive. Just an observation.